src/renderers/dom/shared/hooks/possibleStandardNames.js

```diff
@@ -307,6 +307,190 @@ var possibleStandardNames = {
   xmlspace: 'xmlSpace',
   ychannelselector: 'yChannelSelector',
   zoomandpan: 'zoomAndPan',
+  accept: 'accept',
+  action: 'action',
+  alt: 'alt',
+  as: 'as',
+  async: 'async',
+  capture: 'capture',
+  challenge: 'challenge',
+  checked: 'checked',
+  cite: 'cite',
+  cols: 'cols',
+  content: 'content',
+  controls: 'controls',
+  coords: 'coords',
+  data: 'data',
+  default: 'default',
+  defer: 'defer',
+  dir: 'dir',
+  disabled: 'disabled',
+  download: 'download',
+  draggable: 'draggable',
+  form: 'form',
+  headers: 'headers',
+  height: 'height',
+  hidden: 'hidden',
+  high: 'high',
+  href: 'href',
+  icon: 'icon',
+  id: 'id',
+  integrity: 'integrity',
+  is: 'is',
+  kind: 'kind',
+  label: 'label',
+  lang: 'lang',
+  list: 'list',
+  loop: 'loop',
+  low: 'low',
+  manifest: 'manifest',
+  max: 'max',
+  media: 'media',
+  method: 'method',
+  min: 'min',
+  multiple: 'multiple',
+  muted: 'muted',
+  name: 'name',
+  nonce: 'nonce',
+  open: 'open',
+  optimum: 'optimum',
+  pattern: 'pattern',
+  placeholder: 'placeholder',
+  poster: 'poster',
+  preload: 'preload',
+  profile: 'profile',
+  rel: 'rel',
+  required: 'required',
+  reversed: 'reversed',
+  role: 'role',
+  rows: 'rows',
+  sandbox: 'sandbox',
+  scope: 'scope',
+  scoped: 'scoped',
+  scrolling: 'scrolling',
+  seamless: 'seamless',
+  selected: 'selected',
+  shape: 'shape',
+  size: 'size',
+  sizes: 'sizes',
+  span: 'span',
+  src: 'src',
+  start: 'start',
+  step: 'step',
+  style: 'style',
+  summary: 'summary',
+  target: 'target',
+  title: 'title',
+  type: 'type',
+  value: 'value',
+  width: 'width',
+  wmode: 'wmode',
+  wrap: 'wrap',
+  about: 'about',
+  datatype: 'datatype',
+  inlist: 'inlist',
+  prefix: 'prefix',
+  property: 'property',
+  resource: 'resource',
+  typeof: 'typeof',
+  vocab: 'vocab',
+  color: 'color',
+  results: 'results',
+  security: 'security',
+  unselectable: 'unselectable',
```

Comment:
Yes. It's a combination of the HTML Property Config and SVG Property Config, which where ultimately just blended into the same list inside of `DOMProperty.properties`

Wrestled with some editor-fu, but I've alphabetized these while still clumping together aliases.